scripts/remove-inactive-members.js

```diff
@@ -0,0 +1,85 @@
+#!/usr/bin/env bash
+":" //# comment; exec /usr/bin/env node --input-type=module - "$@" < "$0"
+
+import { writeFileSync, readFileSync } from 'fs';
+
+const __dirname = process.cwd();
+const args = process.argv.slice(2);
+
+const unimportantCategories = ['org_credential_authorization'];
+
+const auditLogFilePath = args[0];
+const auditLog = JSON.parse(readFileSync(auditLogFilePath)).filter(event => {
```

Comment:
Can we attach the "audit log" for reference and/or a note on how it was generated?

Reply:
```suggestion
// You can export the audit log from the organization settings page
// See https://docs.github.com/en/organizations/keeping-your-organization-secure/managing-security-settings-for-your-organization/reviewing-the-audit-log-for-your-organization#exporting-the-audit-log
const auditLog = JSON.parse(readFileSync(auditLogFilePath)).filter(event => {
```

How about adding information where to get the audit log from? 

> For example, this may be more clear for folks when they look at "remove-inactive-members.js". We may want to call that out.

I added a section on `How do I check why I was mentioned?` to the description now.

A permission inspection tool is a pretty cool idea, let's track it here: https://github.com/protocol/github-mgmt-template/issues/39

As for the general approach to permissions strategy, I view this PR as a first step required to get there. Here, we're cleaning up what is not being used to make the reasoning about the current state easier. I created a new issue with your questions in it to make sure we don't loose track of it - https://github.com/libp2p/github-mgmt/issues/14